rebar.config

```diff
@@ -23,10 +23,7 @@
          {git, "git://github.com/opscode/mini_s3.git",
           {branch, "master"}}},
         {eper, ".*",
-         {git, "git://github.com/massemanet/eper.git", {branch, "master"}}},
-        {rebar_lock_deps_plugin, ".*",
-         {git, "git://github.com/seth/rebar_lock_deps_plugin.git", "master"}}
```

Comment:
This is probably my unfamiliarity with relx - does it replace the lock-deps functionality, commit log generation for deps, and tag creation that we used in rebar_lock_deps_plugin?